Contributing.md

```diff
@@ -0,0 +1,5 @@
+# contributing to Sway
+
+We welcome contributions to Sway!
+
+Please see the [Contributing To Sway](https://fuellabs.github.io/sway/latest/reference/contributing_to_sway.html) section of the Sway book for guidelines and instructions to help you get started.
```

Comment:
Is there really a need for a whole new file? Could this be at the end of the readme?